<launch>

	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
		<arg name="port" value="9090"/>
	</include>

	<include file="$(find gatlin)/launch/baxter_tf.launch" />

	<!-- <node pkg="gatlin" type="frame_maintainer.py" name="frame_maintainer" output="screen"/> -->
	
  <!-- <node pkg="tf" type="static_transform_publisher" name="base_link_tf" args="-0.000, -0.692, -0.057 0 0 0 1 /baxter_link /base 100"/> -->

  <!-- <node pkg="gatlin" type="baxter_controller.py" name="baxter_controller" output="screen"/> -->

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find gatlin)/rviz/server.rviz"/>

</launch>